docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1096,3 +1098,55 @@ function su_humsci_profile_update_9718() {
   // Uninstalls modules after removing configs to prevent updb errors.
   \Drupal::service('module_installer')->uninstall(['hs_webform', 'webform_ui', 'webform']);
 }
+
+/**
+ * Add new logout block to existing sites.
+ */
+function su_humsci_profile_update_9719() {
+  $config_storage = new FileStorage(Settings::get('config_sync_directory'));
+  $entity_type_manager = \Drupal::service('entity_type.manager');
+
+  $config_names = [
+    'block.block.humsci_colorful_samlsunetidlogoutblock',
+    'block.block.humsci_traditional_samlsunetidlogoutblock',
+  ];
+
+  foreach ($config_names as $config_name) {
+
+    // Load the configuration data from the sync directory.
+    $config_data = $config_storage->read($config_name);
+
+    if (!$config_data) {
+      throw new \Exception(t('The configuration %config_name was not found in the sync directory.', ['%config_name' => $config_name]));
+    }
+
+    // Get the block storage and possible existing config.
+    $block_storage = $entity_type_manager->getStorage('block');
+    $existing_config = $block_storage->load($config_data['id']);
+
+    if (!$existing_config) {
```

Comment:
isn't it guaranteed there will not be existing config? These are new configs and blocks are prefixed if they are custom on sites.